config/initializers/airbrake.rb

```diff
@@ -1,3 +1,4 @@
 Airbrake.configure do |config|
-  config.api_key = ENV['AIRBRAKE_API_KEY']
+  config.project_key = ENV['AIRBRAKE_API_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.